dll/win32/dbgeng/dbgeng.c

```diff
@@ -0,0 +1,4129 @@
+            void *buffer, ULONG buf_size, const WCHAR *initial, const WCHAR *environment, ULONG processid, ULONG flags)
+{
+    FIXME("%p %s, %s, %p, %d, %s, %s, %d, 0x%08x.\n", iface, wine_dbgstr_longlong(server), debugstr_w(command), buffer,
+            buf_size, debugstr_w(initial), debugstr_w(environment), processid, flags);
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_PushOutputLinePrefix(IDebugClient7 *iface, const char *prefix, ULONG64 *handle)
+{
+    FIXME("%p, %p.\n", iface, handle);
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_PushOutputLinePrefixWide(IDebugClient7 *iface, const WCHAR *prefix, ULONG64 *handle)
+{
+    FIXME("%p, %p.\n", iface, handle);
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_PopOutputLinePrefix(IDebugClient7 *iface, ULONG64 handle)
+{
+    FIXME("%p, %s.\n", iface, wine_dbgstr_longlong(handle));
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_GetNumberInputCallbacks(IDebugClient7 *iface, ULONG *count)
+{
+    FIXME("%p, %p.\n", iface, count);
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_GetNumberOutputCallbacks(IDebugClient7 *iface, ULONG *count)
+{
+    FIXME("%p, %p.\n", iface, count);
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_GetNumberEventCallbacks(IDebugClient7 *iface, ULONG flags, ULONG *count)
+{
+    FIXME("%p, 0x%08x, %p.\n", iface, flags, count);
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_GetQuitLockString(IDebugClient7 *iface, char *buffer, ULONG buf_size, ULONG *size)
+{
+    FIXME("%p, %s, %d, %p.\n", iface, debugstr_a(buffer), buf_size, size);
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_SetQuitLockString(IDebugClient7 *iface, char *string)
+{
+    FIXME("%p, %s.\n", iface, debugstr_a(string));
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_GetQuitLockStringWide(IDebugClient7 *iface, WCHAR *buffer, ULONG buf_size, ULONG *size)
+{
+    FIXME("%p, %s, %d, %p.\n", iface, debugstr_w(buffer), buf_size, size);
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_SetQuitLockStringWide(IDebugClient7 *iface, const WCHAR *string)
+{
+    FIXME("%p, %s.\n", iface, debugstr_w(string));
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_SetEventContextCallbacks(IDebugClient7 *iface, IDebugEventContextCallbacks *callbacks)
+{
+    FIXME("%p, %p.\n", iface, callbacks);
+    return E_NOTIMPL;
+}
+
+static HRESULT STDMETHODCALLTYPE debugclient_SetClientContext(IDebugClient7 *iface, void *context, ULONG size)
+{
+    FIXME("%p, %p, %d.\n", iface, context, size);
+    return E_NOTIMPL;
+}
+
+#ifdef __REACTOS__
+static IDebugClient7Vtbl debugclientvtbl =
+#else
+static const IDebugClient7Vtbl debugclientvtbl =
+#endif
```

Comment:
Why is this required?

Reply:
@learn-more I made this change (and same others in similar structures) to fix some compilation errors. But looks like now, after defining `__WINESRC__` in CMakeLists.txt, they are gone, even when I removed these diffs. So I'll revert them all, if it will not broke the compilation.
Thanks for noticing! :+1:

Reply:
Fixed.
Still compilable for me.